src/main/java/gyro/aws/ec2/PlacementGroupResource.java

```diff
@@ -0,0 +1,188 @@
+package gyro.aws.ec2;
+
+import com.psddev.dari.util.ObjectUtils;
+import gyro.aws.AwsResource;
+import gyro.aws.Copyable;
+import gyro.core.GyroException;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Id;
+import gyro.core.scope.State;
+import gyro.core.validation.ValidationError;
+import software.amazon.awssdk.services.ec2.Ec2Client;
+import software.amazon.awssdk.services.ec2.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Creates a Placement Group with the specified name and placement strategy.
+ *
+ * Example
+ * -------
+ *
+ * .. code-block:: gyro
+ *
+ *     aws::placement-group example-placement-group
+ *         group-name: "TestGroup"
+ *         placement-strategy: "partition"
+ *         partition-count: 3
+ *     end
+ */
+
+@Type("placement-group")
+public class PlacementGroupResource extends Ec2TaggableResource<PlacementGroup> implements Copyable<PlacementGroup> {
+
+    private String groupName;
+    private PlacementStrategy placementStrategy = PlacementStrategy.CLUSTER;
+    private Integer partitionCount;
+    private String id;
+
+    /**
+     * A name to identify the Placement Group. (Required)
+     */
+    public String getGroupName() {
+        return groupName;
+    }
+
+    public void setGroupName(String groupName) {
+        this.groupName = groupName;
+    }
+
+    /**
+     * One of three approaches towards managing the placement of instances on the underlying hardware (cluster | spread | partition).
+     * Defaults to the "cluster" strategy.
```

Comment:
Dont just copy the documentation as is. This is the getter for the finder, so there are no default values but what ever you provide will be used to filter it.
```suggestion
     * Defaults to the "cluster" strategy.
```